input/correu-genkwhAssignacio.mako

```diff
@@ -9,11 +9,20 @@ import datetime
 def slashdate(date):
     return date.strftime('%d/%m/%Y')
 
+md_obj = object.pool.get('ir.model.data')
+gkwh_emission_id = md_obj.get_object_reference(
+                    object._cr, object._uid,  'som_generationkwh', 'emissio_genkwh'
+                )[1]
+
+today = datetime.date.today().strftime('%Y-%m-%d')
```

Comment:
As you said, this may be no necessary. But it only hurts if for whatever reason, the cron won't be executed.
On the other hand, the issue of send a wrong date when Investments are not filtered correctly won't happen again.
Your choice!

Reply:
I decided to remove it, because the mail is only sent for the first investment. Also, without it, we can resend the email anytime and the original date will be displayed.